app/partials/states/account_settings.html

```diff
@@ -295,16 +295,18 @@ <h2 class='subheader'>Editing another user's Isaac account.</h2>
                                     <strong ng-if="!user.hasSegueAccount">Set password</strong>
                                 </label>
                                 <!-- Can't set new password if not confirmed old one, or if don't have account with a password. Unless admin. -->
-                                <input id="account-password" ng-disabled="(!passwordChangeState.passwordCurrent || !user.hasSegueAccount)  && editingSelf" type="password" ng-model="password1" name="password" placeholder="Enter password">        
```

Comment:
Why did this line get deleted? It's surely the important one where you confirm your current password?